### BIG differences, this is diff of wrong files.
### CLUS "System Class RATIONAL"
### and CLHS topic selection http://clhs.lisp.se/Body/a_ration.htm
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:types:rational.txt	2017-04-27 21:49:36.000000000 +0100
+++ saved-pages-clus/cl:types:rational.txt	2017-04-27 21:49:36.000000000 +0100
@@ -1,4 +1,45 @@
-RATIONAL
-Please select which reference to RATIONAL you intended
+System Class RATIONAL

-FunctionSystem ClassGlossary Entry
\ No newline at end of file
+Class Precedence List
+rational, real, number, t
+Description
+The canonical representation of a rational is as an integer if its value is integral, and otherwise as a ratio.
+The types integer and ratio are disjoint subtypes of rational.
+Compound Type Specifier Kind
+Abbreviating.
+Compound Type Specifier Syntax
+ rational [lower-limit [upper-limit]]
+Compound Type Specifier Arguments
+ lower-limit, upper-limit - interval designators for type rational. The defaults for each of lower-limit and upper-limit is the symbol *.
+Compound Type Specifier Description
+This denotes the rationals on the interval described by lower-limit and upper-limit.
+\
\ No newline at end of file
